manifests/java.yml

```diff
@@ -910,8 +910,8 @@ tests/:
       
     test_span_links.py: missing_feature
     test_telemetry.py:
-      Test_Defaults: missing_feature
-      Test_Environment: missing_feature
+      Test_Defaults: v1.28.0
```

Comment:
Java tests currently failing because of two reasons:
1. Naming difference, that relies on [the config norm rules](https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json) applied on the backend. These assertions should probably be relaxed and check the value depending on the tracer, e.g. `logs_injection_enabled` is `logs.injection` in case of Java Tracer as it outlined in [the config norm rules](https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json#L41C3-L41C45).
2. `trace_sample_rate` specifically in Java defaults to `null` instead of `1.0` but it's effectively the same because if it's `null` it would use the special case `AllSampler`.